test/app/redis/node/server.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+app.get('/redis-test', function (req, res) {
+
+	let messages = [];
+
+	var redis = serviceManager.get('redis');
+
+	if (!redis) {
+		res.status(500).send('redis is not defined in serviceManager');
+		return;
+	}
+
+	redis.set("test-key", "test-val");
+
+	messages.push("set data");
+
+	redis.get("test-key", function (err, response) {
```

Comment:
Same here. This doesn't actually validate if the data we just inserted is the same data we're getting back.

Reply:
I followed the pattern that most of the other node and python tests follow. If we want more extensive tests then we'll need to address all services.